Sources/TripKitUI/cards/TKUIRoutingResultsCard.swift

```diff
@@ -632,6 +639,11 @@ private extension TKUIRoutingResultsCard {
     return modePicker
   }
   
+  func repositionErrorView(sittingAbove tableView: UITableView, in cardView: TGCardView) {
+    let parent = (cardView as? TGScrollCardView)?.scrollViewWrapper ?? cardView
```

Comment:
This relies on the internals of TGCardVC. So if anything in there changes, this would break.

A safer approach would be for card to (weakly) hang on on to the instances returned by `TKUITripBoyView.show`, and this method then just passing the new padding to it here.

Reply:
Good point. Done, can you please give it another look?